### ISSUE TITLE
Build looks in the wrong directory for the docbook.xsl

### ISSUE THREAD

**Issue by Earnestly** [dankamongmen/omphalos#4]
Using `./configure LIBSYSFS_CFLAGS=-I/usr/include/sysfs LIBSYSFS_LIBS=-lsysfs CFLAGS-=Wno-error --prefix=/usr` and `make` I am able to build.

However it seems that you look in the wrong location for `docbook.xsl` (I assume). 

The package for [Arch Linux](https://projects.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/docbook-xsl) installs this particular file to `/usr/share/xml/docbook/xsl-stylesheets-1.78.1/xhtml5/docbook.xsl` and `/usr/share/xml/docbook/xsl-stylesheets-1.78.1/manpages/docbook.xsl`

```
echo -e '<!ENTITY dhfirstname "Nick">\n<!ENTITY dhsurname "Black">\n<!ENTITY dhemail "nick.black@sprezzatech.com">\n<!ENTITY dhusername "&dhfirstname; &dhsurname;">\n<!ENTITY dhrelease "0.99.7-rc1">' > "out/doc/xml/omphalos.ent"
/usr/bin/xsltproc --path out/doc/xml --writesubtree out/doc/man/man1 -o out/doc/man/man1/omphalos.1 --nonet /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl doc/man/man1/omphalos.xml
warning: failed to load external entity "/usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl"
cannot parse /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
Makefile:772: recipe for target 'out/doc/man/man1/omphalos.1' failed
make: *** [out/doc/man/man1/omphalos.1] Error 4
make: *** Waiting for unfinished jobs....
```

**Comment by dankamongmen** [comment]
This same issue affects growlight. Time to fix it.

**Comment by dankamongmen** [comment]
https://github.com/Debian/apt/issues/32 here's a similar problem. They suggest running a find, gross.

**Comment by dankamongmen** [comment]
Fuck it, we'll do it live. Dropping the --nonet, and allowing it to be fetched from (gasp) sourceforge.

iff --git a/Makefile.am b/Makefile.am
index 9986387..798f4a5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -33,8 +33,10 @@ DOCS:=$(MAN1OBJ) $(XHTML)
 
 # This can be a URL; it's the docbook-to-manpage XSL
 # Should the network be inaccessible, and local copies are installed, try:
-DOCMANXSL:=--nonet /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
-DOCXHTMLXSL:=--nonet /usr/share/xml/docbook/stylesheet/docbook-xsl/xhtml/docbook.xsl
+#DOCMANXSL:=--nonet /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
+#DOCXHTMLXSL:=--nonet /usr/share/xml/docbook/stylesheet/docbook-xsl/xhtml/docbook.xsl
+DOCMANXSL:=http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
+DOCXHTMLXSL:=http://docbook.sourceforge.net/release/xsl/current/xhtml/docbook.xsl
 
 USBIDS:=usb.ids
 IANAOUI:=ieee-oui.txt

**Comment by dankamongmen** [comment]
Resolved in 0.99.8.
